tests/build/build_templates.go

```diff
@@ -112,6 +113,20 @@ var _ = framework.BuildSuiteDescribe("Build templates E2E test", Label("build",
 		})
 
 		AfterAll(func() {
+			//Remove finalizer from symlink check pipelinerun
```

Comment:
We create a component separately [here](https://github.com/redhat-appstudio/e2e-tests/blob/6fff2460639059b0cb041bacbc37c0c01558ee59/tests/build/build_templates.go#L102-L111), since it is special case and it is not listed in the componentURLs [here](https://github.com/redhat-appstudio/e2e-tests/blob/6fff2460639059b0cb041bacbc37c0c01558ee59/tests/build/const.go#L49) , so it need to be handled seperately.